blog/docs/events/fuzzing-audit-2023.md

```diff
@@ -0,0 +1,39 @@
+# Google Summer of Code 2023 at Knative
```

Comment:
Title should probably be something about fuzzing :smile:

Reply:
Thanks 😄